.changeset/short-cows-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@anonklub/halo2-eth-membership": major
```

Comment:
let's make it a minor version.
We are not adding new features, just change the configuring of what already exists and modifying tests.

Reply:
Done.